paasta_tools/marathon_serviceinit.py

```diff
@@ -103,13 +103,29 @@ def status_desired_state(service, instance, client, job_config):
     return "State:      %s - Desired state: %s" % (status, desired_state)
 
 
+def get_app_queue_status(app_id, client):
```

Comment:
could you move this into marathon_tools please? also, please swap the order of the params to make it ``client, app_id``